lib/dslash_policy.cuh

```diff
@@ -646,7 +646,7 @@ namespace quda
 
             // Query if comms has finished
             if (!pattern.commsCompleted[2 * i + dir]) {
-              if (commsComplete(*in, dslash, i, dir, true, true, false)) {
+              if (commsComplete(in, dslash, i, dir, true, true, false)) {
                 ;
```

Comment:
For the sheer virtue that there's a lot of other cleanup in this PR, can we delete this `;` as part of some other commit?

Reply:
7c4a177f21901ac2003b750df85f9f19e90ce20d